app/src/org/odk/collect/android/activities/FormEntryActivity.java

```diff
@@ -1327,45 +1324,6 @@ private void clearAnswer(QuestionWidget qw) {
         qw.clearAnswer();
     }
 
-
-    /*
-     * (non-Javadoc)
-     * @see android.app.Activity#onCreateContextMenu(android.view.ContextMenu, android.view.View, android.view.ContextMenu.ContextMenuInfo)
-     */
-    @Override
-    public void onCreateContextMenu(ContextMenu menu, View v, ContextMenuInfo menuInfo) {
-        super.onCreateContextMenu(menu, v, menuInfo);
-            menu.add(0, v.getId(), 0, StringUtils.getStringSpannableRobust(this, R.string.clear_answer));
```

Comment:
Leaving in R.string.clear_answer text entry, and all the other corresponding text entries.
They will be rendered unused, but might be useful in the future? Or should I just delete them?